modules/beetmover_scriptworker/manifests/settings.pp

```diff
@@ -30,6 +30,12 @@
                 firefox     => 'mozilla-releng-dep-partner',
             },
 
+            dep_maven_beetmover_aws_access_key_id => secret('dep_maven_beetmover_aws_access_key_id'),
+            dep_maven_beetmover_aws_secret_access_key => secret('dep_maven_beetmover_aws_secret_access_key'),
+            dep_maven_buckets => {
+                geckoview   => 'geckoviewtest',   # TODO replace with bucket owned by CloudOps
```

Comment:
Are we ever going to have a staging bucket provided by CloudOps?

Reply:
Yes, I'd like to have it. The request has been filed in https://bugzilla.mozilla.org/show_bug.cgi?id=1470937